.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
First: A tab could be a different number of columns depending on your environment, but a space is always one column.
Second: You choose as a developer to use 2 or 4 spaces. I recommend 2 spaces because when you get into nested code you will hit the max. 80 columns later.